src/sdam/topology.ts

```diff
@@ -365,7 +365,7 @@ export class Topology extends EventEmitter {
 
       // TODO: NODE-2471
       if (server && this.s.credentials) {
-        server.command('admin.$cmd', { ping: 1 }, err => {
+        server.command('admin.$cmd', { ping: 1, thisPingShouldBeIgnored: 1 }, err => {
```

Comment:
Would love to chat more about this, open to any solutions, Shane and I came up with this one. 

I originally did ignore all ping commands. This was a tricky one to solve because there is a ping check in one of the spec tests: 

https://github.com/mongodb/specifications/blob/master/source/atlas-data-lake-testing/tests/runCommand.yml

That test runs with auth, so we can't implement your comment below:

> it seems like a more general solution would be to just drop the first ping if credentials are provided?

Would love your feedback on how to move forward on this.

Reply:
> I originally did ignore all ping commands.

I was saying if a test runs with auth then we should skip the _first_ recorded ping operation, not all of them. We know that the driver right now will _always_ send an initial `ping` if credentials are provided, so it seems safe for us to ignore them in that case

Reply:
Done!